### DISCUSSION TITLE
怎么用华为昇腾npu加速推理和训练？求教程！如果不支持，该怎么修改？

### DISCUSSION THREAD

**Discussion by yanyecao** [ymcui/Chinese-LLaMA-Alpaca#470] [Q&A]
华为昇腾p910，怎么才能用这个npu来训练和推演？看文档只提到cpu和gpu。求解答！谢谢

**Comment by airaria** [comment]
没有使用过相关硬件，不过我们的模型结构上除了词表大小，和原版LLaMA是一样的，所以建议搜索在昇腾p910部署LLaMA方法，理论上可以用同样的方式部署Chinese-LLaMA
